aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -919,6 +919,41 @@ Tensor _nested_view_from_buffer(
     std::vector<int64_t>(offsets.begin(), offsets.end()));
 }
 
+/**
+ * Create a [B, *, D] nested tensor that is a view of a 2D values tensor
+ * where offsets are the cumsum of the size along the ragged dimension
+ *
+ * @return Returns a nested tensor that
+ * aliases the same storage as values
+ */
+Tensor _nested_from_values_and_offsets(
```

Comment:
I think that this function should be moved to NestedTensorFactories and if it relies on any things in Nt math either move them to header or if we think they are utils move them to Ntutils